deploy/acm/policies/all_policies/policy5-subscriptions.yaml

```diff
@@ -0,0 +1,208 @@
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: operators.coreos.com/v1
+                        kind: OperatorGroup
+                        metadata:
+                            annotations:
+                                ran.openshift.io/ztp-deploy-wave: "2"
+                            name: cluster-logging
+                            namespace: openshift-logging
+                        spec:
+                            targetNamespaces:
+                                - openshift-logging
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: operators.coreos.com/v1alpha1
+                        kind: Subscription
+                        metadata:
+                            annotations:
+                                ran.openshift.io/ztp-deploy-wave: "2"
+                            name: cluster-logging
+                            namespace: openshift-logging
+                        spec:
+                            channel: stable
+                            name: cluster-logging
+                            source: redhat-operators
+                            sourceNamespace: openshift-marketplace
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: v1
+                        kind: Namespace
+                        metadata:
+                            annotations:
+                                ran.openshift.io/ztp-deploy-wave: "2"
+                                workload.openshift.io/allowed: management
+                            name: openshift-local-storage
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: operators.coreos.com/v1
+                        kind: OperatorGroup
+                        metadata:
+                            annotations:
+                                ran.openshift.io/ztp-deploy-wave: "2"
+                            name: openshift-local-storage
+                            namespace: openshift-local-storage
+                        spec:
+                            targetNamespaces:
+                                - openshift-local-storage
+                    - complianceType: musthave
+                      objectDefinition:
+                        apiVersion: operators.coreos.com/v1alpha1
+                        kind: Subscription
+                        metadata:
+                            annotations:
+                                ran.openshift.io/ztp-deploy-wave: "2"
+                            name: local-storage-operator
+                            namespace: openshift-local-storage
+                        spec:
+                            channel: "4.9"
+                            installPlanApproval: Automatic
```

Comment:
I think installPlan will only be approved when its Approval is "Manual"? I know it doesn't really matter of the value of installPlanApproval set here for testing, as you are patching the installPlans with Approval Manual in the created MCVs later,  but I feel a bit confused when seeing the sub has installPlanApproval with Automatic but its installplan finally cat get approved. I think better to provide the example sub yaml that match the logic? Maybe explicitly add the installPlanApproval: Manual in other subs as well because the default is Automatic.